### PR TITLE
Aswathy/Cellxpert personal details

### PR DESCRIPTION
**Pull request by aswathy-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/signup-affiliates/components/_account-personal-details.tsx [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+        certificate,
+        citizen,
+        website_url,
+        social_media_url,
+        password,
+    ])
+
+    useEffect(() => {
+        onValidate(validate)
+    }, [onValidate, validate])
```
Comment (NikitK-deriv) [inline src/pages/signup-affiliates/components/_account-personal-details.tsx:287]: Are we need to add `onValidate ` in dependency of useEffect?

File: src/pages/signup-affiliates/components/_account-personal-details.tsx [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+                    'company_registration_number',
+                ]
+
+                return !company_details.includes(item.name)
+            }
+            return true
+        })
+    }
+    const handleInput = (e) => {
+        console.log(e)
```
Comment (NikitK-deriv) [inline src/pages/signup-affiliates/components/_account-personal-details.tsx:304]: Are we need this?

File: src/pages/signup-affiliates/components/_account-personal-details.tsx [hunk 3]
```diff
@@ -0,0 +1,409 @@
...
+    const [certificate_error_msg, setCertificateErrorMsg] = useState('')
+    const [currency_error_msg, setCurrencyErrorMsg] = useState('')
+
+    const form_inputs = [
+        {
+            id: 'first_name',
+            name: 'first_name',
+            type: 'text',
+            label: localize('First name'),
+            placeholder: 'First name',
```
Comment (NikitK-deriv) [inline src/pages/signup-affiliates/components/_account-personal-details.tsx:115]: Shouldn't we localize placeholder as well for all of this cases?

File: src/pages/signup-affiliates/wizard.tsx [hunk 4]
```diff
@@ -51,6 +63,23 @@ const AffiliateSignup = () => {
...
+                        website_url: value.website_url,
+                        password: value.password,
+                        company_name: value.company_name,
+                        company_registration_number: value.company_registration_number,
+                        certificate: value.certificate,
+                        citizen: value.citizen,
+                    },
+                })
         }
     }
```
Comment (NikitK-deriv) [inline src/pages/signup-affiliates/wizard.tsx:84]: Seems here should be `break` as for other cases above

File: src/pages/signup-affiliates/_currency-select.tsx [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+        <Flex jc="space-evenly" mt="30px" mb="32px">
+            {currency_list.map((currency, idx) => {
+                const selected = current_select === currency.value
+                return (
+                    <Item
+                        fd="column"
+                        ai="center"
+                        id={currency.value}
+                        onClick={() => handleCurrencyData(currency.value)}
+                        key={idx}
```
Comment (prince-deriv) [inline src/pages/signup-affiliates/_currency-select.tsx:76]: This will cause duplicate keys. Combine the `idx` with some descriptive text.

**Pull request closed without merging**